source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Domain/Dtos/ChargePriceCommands/Validation/InputValidation/ChargePriceOperationInputValidationRulesFactoryTests.cs

```
@@ -53,28 +53,6 @@ public class ChargePriceOperationInputValidationRulesFactoryTests
             actualRuleTypes.Should().Contain(expectedRulesTypes);
         }
 
-        [Theory]
-        [InlineAutoMoqData]
-        public void CreateRules_WhenOperationContainsNoPoints_ShouldContainRules(
```

Would it make sense to add a test case to `CreateRules_WhenChargePriceOperation_ShouldContainRules` where an operation contains no point(s)?

---

If that scenario occurs, it should throw an error in the document validation instead, a story have been added to the validation feature